src/shared/auto-order-json-translations.ts

```diff
@@ -0,0 +1,32 @@
+﻿//@ts-nocheck
+//@ts-ignore
+const fs = require('fs');
+const path = require('path');
+const english = require('./translation/translationEnglish.json');
+const german = require('./translation/translationGerman.json');
+
+const orderedEnglish = {};
+Object.keys(english).sort(function (a, b) {
+    return a.toLowerCase().localeCompare(b.toLowerCase());
```

Comment:
Die Funktion doppelt sich in englisch und deutsch. Vllt die Funktion als eine Funktion refactoren und über beide dicts laufen lassen.